src/components/TransactionsList/TransactionsList-container.tsx

```diff
@@ -1,22 +1,18 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
+import { withRouter } from 'react-router-dom';
 import Table from '@material-ui/core/Table';
 import TableBody from '@material-ui/core/TableBody';
 import TableCell from '@material-ui/core/TableCell';
 import TableHead from '@material-ui/core/TableHead';
 import TableRow from '@material-ui/core/TableRow';
 import Grid from '@material-ui/core/Grid/Grid';
-import Tooltip from '@material-ui/core/Tooltip/Tooltip';
-import TableSortLabel from '@material-ui/core/TableSortLabel/TableSortLabel';
-import TableFooter from '@material-ui/core/TableFooter';
-import TablePagination from '@material-ui/core/TablePagination';
 import Typography from '@material-ui/core/Typography';
 
-import TransactionListPagination from './TransactionsListPagination-view';
 import DetailsModal from './TransactionsListDetailsModal-view';
-import { stableSort, getSorting } from '../../helpers/helpers';
-import { HeaderColsInterface, ModalDetailsProps, Order, OrderBy } from './types';
+import { HeaderColsInterface, ModalDetailsProps, TransactionsListProps } from './types';
 import { timestampToDate } from './../../helpers/helpers';
-import { Block } from "../../types";
+import { transactionsListTableStyle } from './TransactionsList-styles';
```

Comment:
This is a hook. A common convention for naming hooks is to use prefix `use` so name it `use TransactionsListTableStyle `